examples/create-submit-job.py

```diff
@@ -30,10 +30,10 @@
 def get_application_id(
     base_api_url="https://armada-k8s.staging.omnivector.solutions/jobbergate",
```

Comment:
isn't it supposed to point to the localhost url deployed by compose?

Reply:
Well, this example pre-dates it. However, I think this could use a little touch-up anyway. I will make those changes.

Reply:
Updated to use with jobbergate-composed.